simulation/lowfi_simulation/build.gradle

```diff
@@ -0,0 +1,189 @@
+                            srcDirs 'src/main/native/include'
+                        }
+                    }
+                }
+                binaries.all {
+                    if (it instanceof SharedLibraryBinarySpec) {
+                        it.buildable = false
+                        return
+                    }
+                    if (it.targetPlatform.architecture.name == 'athena') {
+                        it.buildable = false
+                        return
+                    }
+                    lib project: ':hal', library: 'hal', linkage: 'shared'
+                    lib project: ':wpiutil', library: 'wpiutil', linkage: 'shared'
+                    project(':ni-libraries').addNiLibrariesToLinker(it)
+                }
+            }
+            "${nativeName}"(NativeLibrarySpec) {
+                sources {
+                    cpp {
+                        source {
+                            srcDirs "${rootDir}/shared/singlelib"
+                            include '**/*.cpp'
+                        }
+                        exportedHeaders {
+                            srcDirs 'src/main/native/include'
+                        }
+                    }
+                }
+                binaries.all {
+                    if (it.targetPlatform.architecture.name == 'athena') {
+                        it.buildable = false
+                        return
+                    }
+                    lib project: ':hal', library: 'hal', linkage: 'shared'
+                    lib project: ':wpiutil', library: 'wpiutil', linkage: 'shared'
+                    project(':ni-libraries').addNiLibrariesToLinker(it)
+                }
+            }
+            // By default, a development executable will be generated. This is to help the case of
+            // testing specific functionality of the library.
+            "${nativeName}Dev"(NativeExecutableSpec) {
+                sources {
+                    cpp {
+                        source {
+                            srcDirs 'src/dev/native/cpp'
+                            include '**/*.cpp'
+                            lib library: 'lowfi_sim'
+                        }
+                        exportedHeaders {
+                            srcDirs 'src/dev/native/include'
+                        }
+                    }
+                }
+                binaries.all {
+                    if (it.targetPlatform.architecture.name == 'athena') {
+                        it.buildable = false
+                        return
+                    }
+                    lib project: ':hal', library: 'hal', linkage: 'shared'
+                    lib project: ':wpiutil', library: 'wpiutil', linkage: 'shared'
+                    project(':ni-libraries').addNiLibrariesToLinker(it)
+                }
+            }
+        }
+        binaries {
+            withType(GoogleTestTestSuiteBinarySpec) {
+                if (!project.hasProperty('onlyAthena')) {
```

Comment:
This is already checked at the top level. Did you mean for this `binaries` block to be outside the !onlyAthena check?